packages/cra-template-typescript/template/src/service-worker.ts

```diff
@@ -0,0 +1,62 @@
+/// <reference lib="webworker" />
+// eslint-disable no-restricted-globals
```

Comment:
@ianschmitz, I believe this needs to use the  `/* eslint-disable no-restricted-globals */` syntax in order to apply to the entire file.